README.md

```diff
@@ -3,3 +3,6 @@
 This repository is used to record architectural decisions made by the
 London-based digital team supporting the Offender Management in Custody
 programme.
+
+To understand why we are recording decisions and how we are doing it, please
+see ADR-0001.
```

Comment:
Should these ADR's have a set of known stakeholders?

Reply:
Yes, I think we do need to clarify that - but it's a separate conversation from deciding to record decisions in this way.